src/AuraLockerModule.sol

```diff
@@ -108,4 +119,13 @@ contract AuraLockerModule is
             )
         ) revert TxFromModuleFailed();
     }
+
+    /// @dev The Gnosis Safe version 1.1.1 does not expose directly `isModuleEnabled` method, so we need a workaround
+    function _isModuleEnabled() internal view returns (bool) {
+        address[] memory modules = SAFE.getModules();
```

Comment:
`getModules()` is limited to the first ten modules in the list, but i think that should be fine for now